include/bluetooth/l2cap.h

```diff
@@ -26,21 +26,61 @@
 extern "C" {
 #endif
 
-/** L2CAP header size, used for buffer size calculations */
+/** L2CAP PDU header size, used for buffer size calculations */
 #define BT_L2CAP_HDR_SIZE               4
 
+/** Maximum Transmission Unit (MTU) for an outgoing L2CAP PDU. */
+#define BT_L2CAP_TX_MTU (CONFIG_BT_L2CAP_TX_MTU)
+
+/** Maximum Transmission Unit (MTU) for an incoming L2CAP PDU. */
+#define BT_L2CAP_RX_MTU (CONFIG_BT_BUF_ACL_RX_SIZE - BT_L2CAP_HDR_SIZE)
+
 /** @def BT_L2CAP_BUF_SIZE
  *
- *  @brief Helper to calculate needed outgoing buffer size, useful e.g. for
- *  creating buffer pools.
+ *  @brief Helper to calculate needed buffer size for L2CAP PDUs.
+ *         Useful for creating buffer pools.
+ *
+ *  @param mtu Needed L2CAP PDU MTU.
+ *
+ *  @return Needed buffer size to match the requested L2CAP PDU MTU.
+ */
+#define BT_L2CAP_BUF_SIZE(mtu) BT_BUF_ACL_SIZE(BT_L2CAP_HDR_SIZE + (mtu))
+
+/** L2CAP SDU header size, used for buffer size calculations */
+#define BT_L2CAP_SDU_HDR_SIZE           2
+
+/** Maximum Transmission Unit for an unsegmented outgoing L2CAP SDU.
+ *
+ *  The Maximum Transmission Unit for an outgoing L2CAP SDU when sent without
+ *  segmentation, i.e a single L2CAP SDU will fit inside a single L2CAP PDU.
+ *
+ *  The MTU for outgoing L2CAP SDUs with segmentation is defined by the
```

Comment:
It's a bit funny to see the mix of "Maximum Transmission Unit", "SDU" and "MTU" in these paragraphs. Is there a reason why MTU is (often) expanded to its full name, but SDU isn't?